### DISCUSSION TITLE
Deployment related questions [Jul 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2903] [Discord Threads]
> 🙋‍♂️ **jeanbonbeur**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### u can ask it here  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### ok perfect thks  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### I deployed NocoDB with Docker-compose on a server with Ubuntu 22.04 to connect the application to a postgresql database.   
#### Do I have to fill in the environment variables in the /nocodb/docker-compose/traefik/.env file?   
#### Then for apply the changes, I must execute ?   
#### $ cd nocodb/docker-compose/pg/   
#### $ docker-compose up -d  
#  
> 🙋‍♂️ **wkw**  
#### u may refer to https://github.com/nocodb/nocodb/blob/develop/docker-compose/pg/docker-compose.yml  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### OK and I indicate my environment variables in this file ?  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### Sorry for these basic questions but I'm completely novice...  
#  
> 🙋‍♂️ **wkw**  
#### u can specify your env vars under `environment`  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### ahh ok. As simple as that !  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### and after that I launch these cmd :  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### $ cd nocodb/docker-compose/pg/   
#### $ docker-compose up -d  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### for apply the changes ?  
#  
> 🙋‍♂️ **wkw**  
#### u may try `docker-compose up -d --build`  
#  
> 🙋‍♂️ **jeanbonbeur**  
#### thks a lot, it seems to be good  🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999969759749738549)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
